xblock/test/test_fields.py

```diff
@@ -390,7 +391,6 @@ def test_none(self):
 
     def test_error(self):
         self.assertJSONOrSetTypeError('abc')
-        self.assertJSONOrSetTypeError('')
```

Comment:
Hmm, I didn't realize this would require explicitly changing error behavior here and changing these test cases in particular. If most XBlock field parsing code is assuming that `blah=""` is a type error for a `List` field, then it's probably better to continue to enforce that strictly. And so now that I see this, I think that if we have some case where there's existing malformed XML that we need to support, it's probably better to support that on a case-by-case basis by fixing it in that block's `parse_xml` method.

Reply:
Ah gotcha..  I've done that in [edx-platform#19303 2dfa8d4](https://github.com/edx/edx-platform/pull/19303/commits/2dfa8d4f1270495841d0c2163f1fbb2a6681e5d4).